lib/Mail/Milter/Authentication.pm

```diff
@@ -273,7 +273,9 @@ sub child_init_hook {
            $protocol &= ~SMFIP_NOBODY;
            $protocol &= ~SMFIP_NOHDRS;
            $protocol &= ~SMFIP_NOEOH;
+           $protocol |= SMFIP_HDR_LEADSPC;
```

Comment:
is there a reason this doesn't follow the '&= ~' style of the lines above?

Reply:
The difference is that all of the callbacks are enabled by default (0 means enabled), whereas HDR_LEADSPC is disabled by default (0 means disabled). The existing code first disables all callbacks by settings those bits to 1 (using SMFIP_NONE), and then it enables each one that it actually wants by flipping the bit back to 0 with &= ~nocb. But HDR_LEADSPC starts disabled, so we just need to set that bit to 1 to enable it.